spread.yaml

```diff
@@ -5,14 +5,16 @@ backends:
     systems:
       - ubuntu-18.04
       - ubuntu-20.04
-      - ubuntu-22.04
+        #- ubuntu-22.04
 
 path: /home/ubuntu-image
 
 prepare: |
   apt update
   apt install snapd
+  snap install core20 --channel=latest/edge
   snap install --classic snapcraft
+  unset SHELL
```

Comment:
This I don't really understand. Is this influencing snapcraft's behavior?

Reply:
Yes, unfortunately it is. See https://bugs.launchpad.net/ubuntu/+source/snapcraft/+bug/1928209 for more information.

TL;DR snapcraft's logic for creating the patchelf command on staged binaries is being influenced by that environment variable, and a bug has been submitted.